kernel/rtlil.cc

```diff
@@ -1892,7 +1892,9 @@ DEF_METHOD(Add,      max(sig_a.size(), sig_b.size()), ID($add))
 DEF_METHOD(Sub,      max(sig_a.size(), sig_b.size()), ID($sub))
 DEF_METHOD(Mul,      max(sig_a.size(), sig_b.size()), ID($mul))
 DEF_METHOD(Div,      max(sig_a.size(), sig_b.size()), ID($div))
+DEF_METHOD(Mod2,     max(sig_a.size(), sig_b.size()), ID($mod2))
 DEF_METHOD(Mod,      max(sig_a.size(), sig_b.size()), ID($mod))
+DEF_METHOD(Rem,      max(sig_a.size(), sig_b.size()), ID($mod))
```

Comment:
I don't see the need for adding a "Rem" method for creating `$mod` cells in addition to the "Mod" method.